apps/drec-ui/src/organization/containers/modals/OrganizationModalsCenter.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { FC } from 'react';
+import { OrganizationAlreadyExists } from './OrganizationAlreadyExists';
+import { RegisterThankYou } from './RegisterThankYou';
+
+export const OrganizationModalsCenter: FC = () => {
+    return (
+        <>
+            <OrganizationAlreadyExists />
+            <RegisterThankYou />
+        </>
```

Comment:
There are some missing modals, though they have implementation in modals reducer side, and also should be showed after some data handlers functions: `RoleChangeModal` and `ChangeMemberOrgRoleModal`.
You should either remove references to them throughout UI or add the view implementation for them.

Reply:
Added the other role modals implementation